modules/apps/object/object-rest-test/src/testIntegration/java/com/liferay/object/rest/internal/resource/v1_0/test/OpenAPIResourceTest.java

```diff
@@ -327,6 +345,20 @@ private String _getNestedEntitySchema(
 		return StringUtil.extractLast(nestedEntitySchema, "/");
 	}
 
+	private JSONObject _getOpenAPIJSONObject(String openAPIYAMLString)
```

Comment:
I think we could rename to a better name, what about:

`private JSONObject _toJSONObject(String yamlString) throws Exception {`